src/Region.cpp

```diff
@@ -729,14 +728,13 @@ boundaryType Halfspace::getMaxZBoundaryType() {
 bool Halfspace::containsPoint(Point* point) {
 
   double evaluation = _surface->evaluate(point) * _halfspace;
-  return (evaluation >= 0);
+  return (evaluation > FLT_EPSILON);
```

Comment:
I disagree with that. 
In edge cases, with points at or near intersection of surfaces, we could have that a point has an evaluation smaller than FLT_EPSILON, then it would be impossible to locate it